watson-pizzeria-pod.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Service to expose frontend
+apiVersion: v1
+kind: Service
+metadata:
+  name: pizza-bot
+  labels:
+    app: pizza-bot
+    tier: frontend
+spec:
+  type: NodePort
+  ports:
+  - port: 3000
+  selector:
+    app: pizza-bot
+    tier: frontend
+---
+# Application to deploy
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: pizza-bot
+spec:
+  replicas: 2 # tells deployment to run 2 pods matching the template
+  template: # create pods using pod definition in this template
+    metadata:
+      labels:
+        app: pizza-bot
+        tier: frontend
+    spec:
+      containers:
+      - name: pizza-bot
+        image: scottdangelo/conversation-slots-intro:latest
```

Comment:
should image name be placeholder example?  Are we building images or publishing them?

Reply:
This needs to be specific, since it is used by kubernetes to deploy the image to the pod.